### DISCUSSION TITLE
How Can I save the Media using custom path in WidgetUI Demo?

### DISCUSSION THREAD

**Discussion by Zangana** [Canardoux/flutter_sound#615] [Q&A]
What I want is that to save the media in a specific path then save that path in a Database so, I can play it later. 
I changed recordingFile to a specific path that I want to save the media didn't work, and I made the temFile return only a specific file still didn't work. I don't know where the media is saved. 

Your help would be appreciated.

**Comment by Larpoux** [comment]
Please, create an issue with your question. It helps me to manage this project.

- Android ?
- iOS ?
- Flutter Web ?
